psh/pshapp/pshapp.c

```diff
@@ -1151,6 +1151,8 @@ static int psh_run(int exitable, const char *console)
 			psh_common.exitStatus = err;
 		}
 		else {
+			err = EXIT_FAILURE;
+			psh_common.exitStatus = err;
```

Comment:
FYI: `ksh`, `bash`, `ash`, `zsh` and other shells in case of unknown command return 127 (`echo $?`).

Reply:
Along with `pshapp.c` there is also `psh.c` to be adjusted accordingly to the above changes (so maybe instead of raw `127` provide a `define` or `enum` ?)
https://github.com/phoenix-rtos/phoenix-rtos-utils/blob/86c1dcacbb7dad3779bab4b4f4f9b04a0b07581b/psh/psh.c#L124-L131

btw. execution of eg. /etc/logo should set `126` (file is not executable)  status instead of say `134577680`.
![2022-11-21-185255_685x405_scrot](https://user-images.githubusercontent.com/141153/203131355-95b25472-0ec5-42a2-80e9-31c9eea0995d.png)